addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.py

```diff
@@ -14,7 +14,7 @@ class L10nInWithholdWizard(models.TransientModel):
     def default_get(self, fields_list):
         result = super().default_get(fields_list)
         active_model = self._context.get('active_model')
-        active_ids = self._context.get('active_ids')
+        active_ids = self._context.get('active_ids', [])
```

Comment:
Any idea why you got the error?  Because normally, it should always have active_ids as the wizard should not be launched from nowhere.

Reply:
Hello @jco-odoo , thanks for the review and `r+` 😊

The reason is that error occured for the test case `test_display_name_new_record` . `default_get` is automatically gets called when a new record is getting created , here in test case it iterated over all the models and for the valid one's it tests `display_name` ,`model.onchange({}, [], fields_spec)` seems to simulate the effect of user interacting with form, and during simulation it calls the `default_get` to get the initial values.